debian/pbspro-execution.prerm

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+# postrm script for pbspro
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#        * <postrm> `remove'
+#        * <postrm> `purge'
+#        * <old-postrm> `upgrade' <new-version>
+#        * <new-postrm> `failed-upgrade' <old-version>
+#        * <new-postrm> `abort-install'
+#        * <new-postrm> `abort-install' <old-version>
+#        * <new-postrm> `abort-upgrade' <old-version>
+#        * <disappearer's-postrm> `disappear' <overwriter>
+#          <overwriter-version>
+# for details, see http://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+
+
+case "$1" in
+    purge|remove|upgrade|failed-upgrade|abort-install|abort-upgrade|disappear)
+    if [ -x /sbin/chkconfig -a -x /sbin/service ]; then
+        out=`/sbin/chkconfig --list pbs 2>/dev/null`
+        if [ $? -eq 0 ]; then
+            if [ -x /etc/init.d/pbs ]; then
+                    /etc/init.d/pbs stop
+            else
+                    /sbin/service pbs stop
+            fi
+            /sbin/chkconfig --del pbs
+    fi
+    else
+        [ -x /etc/init.d/pbs ] && /etc/init.d/pbs stop
+        rm -f /etc/rc0.d/K10pbs
+        rm -f /etc/rc1.d/K10pbs
+        rm -f /etc/rc2.d/K10pbs
+        rm -f /etc/rc3.d/S90pbs
+        rm -f /etc/rc4.d/K10pbs
+        rm -f /etc/rc5.d/S90pbs
+        rm -f /etc/rc6.d/K10pbs
+    fi
+    rm -f /opt/pbs/etc/db_user.new
```

Comment:
@mike0042, thanks for your time for the review. This is a debian maintainer script and is equivalent to %preun of RPM world. This should remain. After installation this will be placed under /var/lib/dpkg/info and named as pbspro-execution.prerm.